isaqb-org/pdf-theme
# Add new default font

When these changes are merged into master, we have to adjust the `.gradle` files in all repositories. We need to add a new attribute to the task definition:

`  'pdf-fontsdir'     : '../pdf-theme/fonts',`

To test the changes:
- checkout the branch in your submodule
- add the attribute to the render task (where `pdf-style` is defined)
- render PDF files and check if everything went well

Fixes #11.
